modules/messages/tests/server/message-to-influx-functional.server.service.tests.js

```diff
@@ -20,8 +20,9 @@ var reachEventEmitter = new EventEmitter();
 
 // mocking the influxdb.writeMeasurement()
 var influxdb = {
-  writeMeasurement: function (measurement, fields, tags) {
+  writeMeasurement: function (measurement, fields, tags, callback) {
     reachEventEmitter.emit('reachedInfluxdb', measurement, fields, tags);
+    if (callback) callback();
```

Comment:
Is this causing tests to hang up? https://travis-ci.org/Trustroots/trustroots/builds/179439231

Removing it yelds some errors, tho.